packages/create-app/templates/default-app/app-config.development.yaml

```diff
@@ -1,8 +1,8 @@
 app:
-  baseUrl: http://localhost:3000
+  baseUrl: http://0.0.0.0:3000
```

Comment:
These should stay as `localhost`

Reply:
Setting this to `localhost` prevents it from being exposed outside of the docker container. I think the app-plugin could be a good alternative to running two servers. Not sure if that will be made the standard.

Reply:
The default is to bind to `::` and `0.0.0.0`, but that's a different configuration under `backend.listen`. This is the URL that callers should use to reach the service, so for development it's best to keep as `localhost`. For production deployments you'd set this to the public URLs of the frontend and backend respectively, which may be the same URL.

Reply:
Thanks for clarifying